src/components/advanced-color-picker/advanced-color-picker.stories.mdx

```diff
@@ -76,3 +77,25 @@ Shows how the color picker can be used to select a color from a predefined set.
 ### AdvancedColorPicker
 
 <StyledSystemProps of={AdvancedColorPicker} noHeader margin />
+
+## Translation keys
+
+The following keys are available to override the translations for this component by passing in a custom locale object to the
+[i18nProvider](https://carbon.sage.com/?path=/story/documentation-i18n--page).
+
+<TranslationKeysTable
```

Comment:
I know you mentioned this to me yourself but just to make sure it's in writing - you should include the `ariaLabel` key in here as well.